app/spell_check.py

```diff
@@ -9,6 +10,17 @@ def parse_txt(raw_input):
             word_list.append(word)
     return word_list
 
+def parse_txt_other_lang(raw_input):
+    init_word_list = re.split(' ', raw_input)
+    word_list = []
+    for word in init_word_list:
+        if word and word != ' ':
+            if word[-1] in '\.\?!,;_:%“”\' …\+"’‘':
```

Comment:
Why are we only looking for these non alphanumeric characters at the last character of the 'words'?

Do we want words with non alphanumeric characters in them? For instance, an email address or website url?
This isn't necessarily wrong, we should just align it with our english parser so that they identify similar items as words or non words.

Reply:
This parse function is parsing the data from the front end, and was built because the English parser was splitting at the diacritics. 
I figured these are just the most common special characters that would appear at the end of the word. As far as email addresses or website urls we can either just let it flag those as misspelled, or add something in to except it

Reply:
okay, figured out what was going on here. The special characters at the end of the word could be legitimate punctuation and needs split from the word and recombined later, whereas special characters in the word will either be a legitimate hyphen, or a misspelling so I basically built this to acknowledge words with special characters as misspelled unless they appear in the corpus

Reply:
Taken care of by other PR in development

Reply:
Taken care of by reworking Regex PR